pkgs/by-name/gi/git-lfs/package.nix

```diff
@@ -71,6 +102,8 @@ buildGoModule rec {
     updateScript = nix-update-script { };
   };
 
+  __darwinAllowLocalNetworking = true;
```

Comment:
Are there any remaining tests that need this, with the networking tests being disabled above?

Reply:
Yes, it fails with another error if I remove it.